Mathlib/Data/Matrix/Block.lean

```diff
@@ -438,7 +438,7 @@ theorem blockDiagonal_pow [DecidableEq m] [Fintype o] [Fintype m] [Semiring α]
   map_pow (blockDiagonalRingHom m o α) M n
 
 @[simp]
-theorem blockDiagonal_smul {R : Type*} [Monoid R] [AddMonoid α] [DistribMulAction R α] (x : R)
+theorem blockDiagonal_smul {R : Type*} [Zero α] [SMulZeroClass R α] (x : R)
```

Comment:
I thought you already generalized this file; did this miss the previous PR?

Reply:
If you look at the most recent commit to the linter, you'll see that the most recent change is just me adding some additional generalizations to Module/DistribMulAction/related SMul classes (e.g. ones that are not child-parent relationships but are still generalizations), and I found those based on your feedback of an additional missing generalization from the last review.